### ISSUE TITLE
Dependency to hamcrest-all not marked as test in spring-content-rest

### ISSUE THREAD

**Issue by nuvitong** [paulcwarren/spring-content#23]
Hi,

in spring-content-rest the dependency to org.hamcrest:hamcrest-all:1.3 is not marked with scope `test`:

```xml
<dependency>
    <groupId>org.hamcrest</groupId>
    <artifactId>hamcrest-all</artifactId>
    <version>1.3</version>
</dependency>
```

Is this on purpose?

Kind regards 

Tobias

**Comment by paulcwarren** [comment]
Probably not Tobias.  Will fix.

**Comment by nuvitong** [comment]
Thanks.

**Comment by paulcwarren** [comment]
Fixed in commit e0a3e377379aa0e28c8f04917388398c467d5a95
